conda/pytorch_cnn.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+echo 'Date: ' `date` 
+echo 'Host: ' `hostname` 
+echo 'System: ' `uname -spo` 
+echo 'GPU: ' `lspci | grep NVIDIA`
+
+# Prepare the dataset
+tar zxf MNIST_data.tar.gz
+
+# Following the example from http://chtc.cs.wisc.edu/conda-installation.shtml
+# except here we download the installer instead of transferring it
+# Download a specific version of Miniconda instead of latest to improve
+# reproducibility
+export HOME=$PWD
+wget -q https://repo.anaconda.com/miniconda/Miniconda3-py37_4.8.2-Linux-x86_64.sh -O miniconda.sh
+sh miniconda.sh -b -p $HOME/miniconda3
+rm miniconda.sh
+export PATH=$HOME/miniconda3/bin:$PATH
+
+# Update conda as workaround for https://github.com/conda/conda/issues/9681
+# Will no longer be needed once conda >= 4.8.3 is available from repo.anaconda.com
+conda install conda=4.8.3
+
+# Set up conda
+source $HOME/miniconda3/etc/profile.d/conda.sh
```

Comment:
does this work? I've had `source` fail on non-interactive jobs...

Reply:
It worked on the batch jobs I submitted to test this.  I'd be curious to learn under which conditions it fails but expect that is hard to determine.

What fails?  The `source` command doesn't exist?

Reply:
yes, the `source` command doesn't exist.

Reply:
@ChristinaLK can you describe more? 
There's differential behavior (`source` availability) in CHTC? (If so, then we should address it, but keep in mind Flocking/Glidein implications.) 
Is it a difference in shell language (e.g. bash vs. sh)? 
Can we get around it with `getenv = true`?

Reply:
I'm happy to let this be as is and we can fix later.

Reply:
Sounds good to me.  If `source` causes an issue later, I'm willing to rewrite this to use a workaround.